src/ROUTES.js

```diff
@@ -96,4 +96,22 @@ export default {
             isParticipantsRoute: Boolean(lodashGet(pathSegments, 2)),
         };
     },
+
+    /**
+     * React Navigation is failing to parse urls with dot in the path segment.
+     * This method enable pasring of urls with dot in path Segment
+     * @param {String} path - Path for the config
+     * @returns {Object}
+     */
+    getEmailRouteLinkingConfig: path => ({
```

Comment:
In the PR details you explain why you moved away from the initial solution but i don't quite understand the current solution. Can you add it to your PR or explain it to me.

Reply:
@chiragsalian  We can customize the default param parsing by using the parse and stringify option for LinkingConfig.
1. In stringify, I replace all the dots with `@dot@` safely. we first encode the existing `@dot@` in the URL and then do the dot conversion. Which will allow `@dot@` to be part of the email.
2. In parse, we revert to the original email by first converting `@dot@` back to dot and then replace the escaped version of `@dot@` back to `@dot@`.

Reply:
Any thoughts @chiragsalian.

Reply:
Ah nice, i had no idea. I just read more about this [here](https://reactnavigation.org/docs/configuring-links/#passing-params) too. Well it makes sense to me and i think its fine, but i'll loop in @marcaaron too for additional thoughts.

Reply:
@marcaaron Do you have any thoughts on this issue?

Reply:
Hey @marcaaron What do you think about it? Sorry for the ping.